packages/design/README.mdx

```diff
@@ -77,3 +77,24 @@ module.exports = {
   ],
 };
 ```
+
```

Comment:
Consider giving more of a description at the top of this file for mobile-minded consumers. This is a h3 heading buried in the doc, but I don't think everything included for mobile is really covered by the top comment. I wonder if this set of instructions should be more prominent... ?

Reply:
Fair. But, is this "Foundational colors and rhythm for the Jobber Atlantis Design System" still accurate or are we adding more to it now?
Also, the "installation" and "usage" at the top are fairly web-centric, aren't they? Should we adjust or clarify that those are for web?

Reply:
> Fair. But, is this "Foundational colors and rhythm for the Jobber Atlantis Design System" still accurate or are we adding more to it now?

I think we should probably update it to be `Foundational colors and styling for the Jobber Atlantis Design System`  I will update that portion to reflect this

>Also, the "installation" and "usage" at the top are fairly web-centric, aren't they? Should we adjust or clarify that those are for web?

I feel like with the section for React Native it implies that it is web since it refers to `CSS` and `PostCSS` to me that heavily implies web implicitly

Reply:
Ok, works for me. :)